lightning/src/ln/channelmanager.rs

```diff
@@ -11049,6 +11049,29 @@ mod tests {
 			sender_intended_amt_msat - extra_fee_msat, 42, None, true, Some(extra_fee_msat)).is_ok());
 	}
 
+	#[test]
+	fn test_final_incorrect_cltv(){
```

Comment:
This still passes if I revert your change above

Reply:
Fixed with the latest changes